frescobaldi_app/vcs/gitdoc.py

```diff
@@ -298,6 +298,7 @@ def set_status(gitprocess, exitcode):
 
         # arguments to get file status
         args = [
+            # TODO: really? including ignored files?
```

Comment:
> At that time I was think that ignored files are a part of the repository and users may open them in Frescobaldi.

Well, that's true (although unlikely). But if a file is ignored we don't need to bother with its diffs, as they won't go into a commit anyway.

> This will also help us deal with the ignored files. 
Do you mean you collect this list in order to *not* check files that are in it? That makes sense, of course